### PR TITLE
[SQL transport 3.1] Updated the connection-settings to include 3.1 changes in pool size warning

### PR DESCRIPTION
**Pull request by SzymonPobiega**
To be merged when SQL transport 3.1 is released.

### CODE REVIEW

File: nservicebus/sqlserver/connection-settings_pool-size_sqlserver_[3,3.1).partial.md [hunk 1]
```diff
@@ -0,0 +1 @@
+NOTE: If the maximum and minimum pool sizes are not explicitly set on the connection string a warning message will be logged. See also [Tuning endpoint message processing](/nservicebus/operations/tuning.md)
```
Comment (SimonCropp) [inline nservicebus/sqlserver/connection-settings_pool-size_sqlserver_[3,3.1).partial.md]: @SzymonPobiega partials have to map to the snippets dirs. ie you would need a 3.1 dir here https://github.com/Particular/docs.particular.net/tree/master/Snippets/SqlServer

**Comment by weralabaj** [comment]
@Particular/sqlserver-transport-maintainers do you still plan on releasing that?

**Comment by SzymonPobiega** [comment]
@weralabaj yes. We're slowly progressing towards 3.1

**Comment by weralabaj** [comment]
rebased @SzymonPobiega that should be merged now correct?

**Comment by SzymonPobiega** [comment]
Yeah 🎉

**Comment by SimonCropp** [comment]
LGTM

**Pull request merged**